portfolio/src/main/webapp/script.js

```diff
@@ -12,37 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
-function getRandomImage(){
+function getRandomImage() {
     const imageId = Math.floor(Math.random()*5) + 1;
     const imageTag = document.getElementById('randomImg');
 
-    imageTag.src = "images/ig"+imageId.toString()+".png";
+    imageTag.src = "images/ig" + imageId.toString() + ".png";
 }
 
-function popupDetail(id){
+function popupDetail(id) {
     const detail = document.getElementById('detail-'+id.toString());
     const button = document.getElementById('button-'+id.toString()); 
     
-    if(button.innerHTML == "Show Details"){
+    if (button.innerHTML == "Show Details") {
         detail.style.visibility = "visible";
         button.innerHTML = "Hide Details";
-    }else{
+    } else {
         detail.style.visibility = "hidden";
         button.innerHTML = "Show Details";
     }
+}
+
+function createListElement(text) {
+    const liElement = document.createElement('li');
+    liElement.innerText = text;
+    return liElement;
+}
+
+async function loadComments() {
+    const response = await fetch("/comments");
+    var comments = await response.text();
+    comments = JSON.parse(comments);
+
+    const commentsListElement = document.getElementById('comments');
+    commentsListElement.innerHTML = '';
+    
+    for (var i=0; i<comments.length; i++) {
```

Comment:
whitespaces: i = 0, i < comments.length, "Comment" + i

Reply:
Sorry for other mistakes. I hope linting will work after merging.